src/gpodder/my.py

```diff
@@ -202,6 +202,11 @@ def __init__(self, config):
         self._worker_thread = None
         atexit.register(self._at_exit)
 
+        # Old versions could set uid to an empty string which causes mygpo to always fail
+        if self._config.mygpo.device.uid == '':
+            self._config.mygpo.device.uid = util.get_hostname_uid()
```

Comment:
```suggestion
            # conform to mygpo device uid limit
            # (https://github.com/gpodder/mygpo/blob/master/mygpo/users/models.py#L235)
            self._config.mygpo.device.uid = util.get_hostname_uid()[:64]
```

Reply:
Should we instead trim the hostname inside `get_hostname_uid()` to `63-len(uid)`? That way we don't trim off the random uid and get long hostname that might be the same across all of a user's devices.

Reply:
yes